frappe_private_comment/overrides/whitelist/comment.py

```diff
@@ -82,3 +84,40 @@ def get_comment_visibility(name: str):
         return None
 
     return {"custom_visibility": doc.custom_visibility}
+
+
+@frappe.whitelist()
+def get_comment_replies(reference_doctype: str, comment_id: str):
+    """Get comment replies"""
+    return frappe.get_all(
+        "Comment",
+        filters={
+            "reference_doctype": reference_doctype,
+            "custom_reply_to": comment_id,
+        },
+        fields=["name", "content", "comment_by", "creation"],
+    )
+
+
+@frappe.whitelist()
+def get_all_replies(reference_doctype: str, reference_name: str):
+    """Get all replies for a comment in a structured format"""
+    replies = frappe.get_all(
```

Comment:
Create one utils function to fetch replies

Reply:
Update: After discussion, for now this is now being used at this api only. Not abstracting for now.